build-support/bin/packages.py

```diff
@@ -305,13 +343,97 @@ def list_packages() -> None:
     print("\n".join(package.name for package in sorted(all_packages())))
 
 
+class PrebuiltWheel(NamedTuple):
+    path: str
+    url: str
+
+    @classmethod
+    def create(cls, path: str) -> "PrebuiltWheel":
+        return cls(path, quote_plus(path))
+
+    def format(self) -> str:
+        return f"{self.path}\t{self.url}"
+
+
+def determine_prebuilt_wheels() -> List[PrebuiltWheel]:
+    """List wheels as tab-separated tuples of filename and URL-encoded name."""
```

Comment:
Could update.

Reply:
Wonderful. Changes to only be the URL, and for the URL to be fully formed, i.e. include the domain prefix. Much better. Thanks!